timesketch/frontend-ng/src/views/Explore.vue

```diff
@@ -516,6 +516,9 @@ limitations under the License.
           </v-btn>
           <!-- Tag menu -->
           <ts-event-tag-menu :event="item"></ts-event-tag-menu>
+          <v-btn small icon @click="addEventBtn(item._source.datetime)">
```

Comment:
This is the button to insert a manual event related to an existing event. With the previous PRs it was accidentally removed.